src/Orleans/Streams/Providers/IStreamProviderRuntime.cs

```diff
@@ -91,20 +91,25 @@ public class PersistentStreamProviderConfig
         public static readonly TimeSpan DEFAULT_STREAM_INACTIVITY_PERIOD = TimeSpan.FromMinutes(30);
 
         public const string QUEUE_BALANCER_TYPE = "QueueBalancerType";
-        public const StreamQueueBalancerType DEFAULT_STREAM_QUEUE_BALANCER_TYPE = StreamQueueBalancerType.ConsistentRingBalancer;
+        //default balancer type if ConsistentRingBalancer
+        public static Type DEFAULT_STREAM_QUEUE_BALANCER_TYPE = null;
```

Comment:
mainly for less breaking change, so user can still do 
`providerSettings.BalancerType = PersistentStreamProviderConfig.DEFAULT_STREAM_QUEUE_BALANCER_TYPE`, when they configure stream provider. 
Although this seems like an unusual usage pattern, since without this line it will still be the default value. So I don't have strong option against removing this